src/biokbase/narrative/tests/narrative_mock/mockclients.py

```diff
@@ -244,7 +245,13 @@ def list_objects(self, params):
     # ----- Execution Engine (EE2) functions -----
 
     def check_workspace_jobs(self, params):
-        return self.job_state_data
+        ee2_states = self.job_state_data
+        if params.get("exclude_fields"):
+            for ee2_state in ee2_states.values():
+                trim_ee2_state(ee2_state, params["exclude_fields"])
+        if params.get("return_list"):
+            ee2_states = list(ee2_states.values())
```

Comment:
Do any of those params ever change? There's only one place where `check_workspace_jobs` gets called, and the params are always the same, so...

Reply:
No, but I thought it might be a good idea to implement the `"exclude_fields"` param since here I'm paying closer attention to when state updates are triggered

Reply:
has adding the `exclude_fields` param changed the output of the function?

Reply:
Today's "good idea to implement" is tomorrow's "why on earth did someone write this?". YAGNI. 😄